crates/matrix-sdk/src/sliding_sync/list/mod.rs

```diff
@@ -231,8 +231,8 @@ impl SlidingSyncList {
     pub(super) fn handle_response(
         &mut self,
         maximum_number_of_rooms: u32,
-        ops: &Vec<v4::SyncOp>,
-        updated_rooms: &Vec<OwnedRoomId>,
+        ops: &[v4::SyncOp],
+        updated_rooms: &[OwnedRoomId],
```

Comment:
There is a clippy lint for this, I wonder why it didn't fire.
Actually, to answer my own question: It's probably `#[instrument]` that confuses clippy about what is macro output vs. code written by hand so it avoids emitting too many lints.